azure/template.json

```diff
@@ -118,7 +118,19 @@
     "appServicePlanInstances": {
       "type": "int",
       "defaultValue": 1
-    }
+    },
+    "logstashHost": {
+      "type": "string",
+      "metadata": {
+        "description": "The host name of the logstash server."
+      }
+    },
+    "logstashPort": {
+      "type": "string",
+      "metadata": {
+        "description": "The port number for the logstash server connection."
+      }
+    },
```

Comment:
@rizzkhan1 Is the trailing comma here a problem?

Reply:
Yes, You don't need a comma in line 133

Reply:
Fixed.